### ISSUE TITLE
Links not working

### ISSUE THREAD

**Issue by nitisht** [miniohq/gluegun#26]
From user `Andrew` on Slack channel:

Hi, the links to `Client grants`, `Configuring wso2`, `Configuring opa`, `Configuring etcd (optional needed only in gateway or federation mode)` in the document page for Minio STS Quickstart Guide (`https://docs.minio.io/docs/minio-sts-quickstart-guide.html`) are not accessible, can someone take a look?

**Comment by kaankabalak** [comment]
For some reason looks like the Github links on the readme were converted to `docs.minio.io` links in the document, will take a look at this.

**Comment by kaankabalak** [comment]
This issue is being caused because the links in the corresponding document are given as follows. These links are being parsed as they are without any changes causing the path to be added to `docs.minio.io/docs/`.
```
## Identity Federation
[**Client grants**](./client-grants.md)
```

```
- [Configuring wso2](./wso2.md)
- [Configuring opa](./opa.md)
- [Configuring etcd (optional needed only in gateway or federation mode)](./etcd.md)
```

In contrast, links to Github that work are given as follows (Taken from [Golang Client Quickstart Guide](https://docs.minio.io/docs/golang-client-quickstart-guide.html) section):
```
## Contribute
[Contributors Guide](https://github.com/minio/minio-go/blob/master/CONTRIBUTING.md)
```

I think we can change that section in the corresponding document to look like the second example to fix this issue.

**Comment by kaankabalak** [comment]
I have sent [this PR](https://github.com/minio/minio/pull/6692) to change the relevant link paths, this should fix the issue.

**Comment by kaankabalak** [comment]
This issue should be fixed now as minio/minio#6692 is merged
